WORKSPACE

```diff
@@ -67,8 +67,9 @@ iap_jwt_verify_nginx_repositories(True)
 
 git_repository(
     name = "com_github_grpc_grpc",
-    commit = "61e77cb7644ef22449fdf7c0ddd7edfe0ae1c9b2",  # v1.21.0
-    remote = "https://github.com/grpc/grpc.git",
+    commit = "edc506849f3429c6b4a6d90ec604aeb1b7b0eb54",  # v1.21.0
+    # TODO: use forked one with some small changes.
+    remote = "https://github.com/qiwzhang/grpc.git",
```

Comment:
What's the change? I think it would be helpful if the comment explains why the change was needed.

Reply:
Added grpc pr number.  Tried to submit these changes to grpc upstream